tests/src/test/java/alluxio/server/ft/journal/raft/EmbeddedJournalIntegrationTestResizing.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.server.ft.journal.raft;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileSystem;
+import alluxio.conf.PropertyKey;
+import alluxio.grpc.NetAddress;
+import alluxio.grpc.QuorumServerInfo;
+import alluxio.grpc.QuorumServerState;
+import alluxio.master.journal.JournalType;
+import alluxio.multi.process.MasterNetAddress;
+import alluxio.multi.process.MultiProcessCluster;
+import alluxio.multi.process.PortCoordination;
+import alluxio.util.CommonUtils;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class EmbeddedJournalIntegrationTestResizing extends EmbeddedJournalIntegrationTestBase {
+
+  @Test
+  public void resizeCluster() throws Exception {
+    final int NUM_MASTERS = 5;
+    final int NUM_WORKERS = 0;
+    mCluster = MultiProcessCluster.newBuilder(PortCoordination.EMBEDDED_JOURNAL_RESIZE)
+        .setClusterName("EmbeddedJournalResizing")
```

Comment:
this name isn't the same format at the other class. Intentional?

Reply:
fixed